build_modules/lib/src/workers.dart

```diff
@@ -66,8 +69,14 @@ BazelWorkerDriver get _analyzerDriver {
   _analyzerWorkersAreDoneCompleter ??= Completer<Null>();
   return __analyzerDriver ??= BazelWorkerDriver(
       () => Process.start(
-          p.join(sdkDir, 'bin', 'dartanalyzer$_scriptExtension'),
-          ['--build-mode', '--persistent_worker'],
+          p.join(sdkDir, 'bin', 'dart'),
+          [
+            p.join(sdkDir, 'bin', 'snapshots', 'dartanalyzer.dart.snapshot'),
```

Comment:
the fact that we're also running from snapshots now needs to be called out in the commit message

Reply:
Done.